tests/logger_loguru/test_attributes.py

```diff
@@ -53,7 +53,7 @@ def _patcher(d):
         bound_logger.error("context_attrs: {}", "arg1", kwarg_attr=4)
 
 
-@validate_log_events([{"message": "exc_info"}], required_attrs=["context.exception"])
+@validate_log_events([{"message": "exc_info"}], required_attrs=["context.file"])
 @validate_log_event_count(1)
 @background_task()
 def test_loguru_exception_context_attributes(logger):
```

Comment:
If this test is for exception context attributes, why are we not checking the exception attribute? Shouldn't that be set?